Source/Dafny/Verifier/Translator.cs

```diff
@@ -6552,7 +6571,7 @@ private class WFOptions {
             var toBound = BaseTypes.BigNum.FromBigInt(BigInteger.One << toWidth); // 1 << toWidth
             var bound = BplBvLiteralExpr(tok, toBound, expr.Type.AsBitVectorType);
             var boundsCheck = FunctionCall(expr.tok, "lt_bv" + fromWidth, Bpl.Type.Bool, o, bound);
-            builder.Add(Assert(tok, boundsCheck, $"{errorMsgPrefix}bit-vector value to be converted might not fit in {toType}"));
+            builder.Add(Assert(tok, boundsCheck, new DafnyConversionFitDescription("bit-vector value", toType.ToString(), errorMsgPrefix)));
```

Comment:
Can we delay the conversion to a string?

Reply:
Yeah, we could. I'll do that. I think there are also other cases where it would make more sense to pass a structured object instead of a string into these descriptions, and I'd love to move toward doing more of that.